src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -152,7 +154,7 @@ public static void SendTimeout_For_Long_Running_Operation_Throws_TimeoutExceptio
             {
                 IWcfService proxy = factory.CreateChannel();
                 watch.Start();
-                proxy.EchoWithTimeout("Hello");
+                proxy.EchoWithTimeout("Hello", serviceOperationTimeout);
             });
         }
         finally
```

Comment:
Is there a better way to check for this? I think both @roncain  and I hate that we are checking against a range of values (prone to issues like this), but I haven't been able to come up with a nicer way to detect that this timeout fired. Thoughts?